### PR TITLE
F4BY target

### PR DESCRIPTION
**Pull request by SergDoc**
add new target - F4BY 
http://swift-flyer.com/
http://hobby.msdatabase.ru/project-updates/theresultsofoursurvey/f4by
http://forum.rcdesign.ru/f123/thread456525.html
http://forum.apmcopter.ru/forums/f4by/

### CODE REVIEW

**Comment by digitalentity** [comment]
@SergDoc please note that PRs in INAV should be done to `development` branch.

File: src/main/target/F4BY/target.h [hunk 1]
```diff
@@ -0,0 +1,170 @@
...
+#define MAG
+#define USE_MAG_HMC5883
+#define MAG_HMC5883_ALIGN       CW90_DEG
+
+#define BARO
+#define USE_BARO_MS5611
+
+#define SENSORS_SET             (SENSOR_ACC|SENSOR_MAG|SENSOR_BARO)
+
+#define OSD
```
Comment (digitalentity) [inline src/main/target/F4BY/target.h]: As far as I know F4BY doesn't have a MAX7456 board

File: src/main/target/F4BY/target.h [hunk 2]
```diff
@@ -0,0 +1,170 @@
...
+#define USE_SPI_DEVICE_3
+#define SPI3_NSS_PIN            PA15
+#define SPI3_SCK_PIN            PB3
+#define SPI3_MISO_PIN           PB4
+#define SPI3_MOSI_PIN           PB5
+
+
+#define USE_I2C
+#define I2C_DEVICE              (I2CDEV_2)
+#define USE_I2C_PULLUP
```
Comment (digitalentity) [inline src/main/target/F4BY/target.h]: This is enabled by default in `common.h`

File: src/main/target/F4BY/target.h [hunk 3]
```diff
@@ -0,0 +1,170 @@
...
+#define RSSI_ADC_PIN            PC1
+
+#define DEFAULT_FEATURES        (FEATURE_VBAT | FEATURE_BLACKBOX)
+#define DEFAULT_RX_FEATURE      FEATURE_RX_PPM
+
+#define ENABLE_BLACKBOX_LOGGING_ON_SDCARD_BY_DEFAULT
+
+
+#define GPS
+#define GPS_PROTO_UBLOX_NEO7PLUS
```
Comment (digitalentity) [inline src/main/target/F4BY/target.h]: This is not safe for some Chinese GPS modules

File: src/main/target/F4BY/target.h [hunk 4]
```diff
@@ -0,0 +1,170 @@
...
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with Cleanflight.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#pragma once
+#define TARGET_BOARD_IDENTIFIER "F4BY"
+
+#define CONFIG_START_FLASH_ADDRESS (0x08080000) //0x08080000 to 0x080A0000 (FLASH_Sector_8)
```
Comment (digitalentity) [inline src/main/target/F4BY/target.h]: This is calculated automatically in INAV

**Review by digitalentity — CHANGES_REQUESTED** [review]

**Comment by SergDoc** [comment]
Ок, no problem, tomorrow...

**Review by digitalentity — APPROVED** [review]

**Pull request merged**